eng/versioning/version_client.txt

```diff
@@ -33,7 +33,7 @@ com.azure:azure-data-appconfiguration;1.1.6;1.2.0-beta.1
 com.azure:azure-data-schemaregistry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.3
-com.azure:azure-digitaltwins-core;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-digitaltwins-core;1.0.0-beta.3;1.0.0
```

Comment:
Are we planning to do a GA release for Digitial twins?

Reply:
Soon, yes. I'm bumping the version now so that the API reviews of this jar don't have this -beta tag in them

Reply:
End of October. We'll be reviewing the GA impl with Azure SDK board again.